docs/en/deploy/Running-Alluxio-Using-Vagrant.md

```diff
@@ -91,35 +187,18 @@ The security group will be set up automatically in the region with all inbound/o
 traffic opened. You can change the *security group*, *region* and *availability zone* in `ec2.yml`. Sometimes the default zone can be unavailable.
 Note: the keypair is associated with a specific region. For example, if you created a keypair in us-east-1, the keypair is invalid in other regions (like us-west-1).  If you ran into permission/connection errors, please first check the region/zone.
 
-**Spot instances**
 
-Using spot instances is a way to reduce EC2 cost. Spot instances are non-guaranteed instances which are priced with bidding.
-Note that spot instances may be taken away from you if someone bids more, and there are no more spot instances available.
-However, for short-term testing, spot instances are very appropriate, because it is rare that spot instances are taken from you.
-
-By default, the deploy scripts DO NOT use spot instances. Therefore, you have to enable deploy scripts to use spot instances.
-
-In order to enable spot instances, you have to modify the file: `deploy/vagrant/conf/ec2.yml`:
-
-    Spot_Price: “X.XX”
-
-For AWS EC2, the default underfs is S3. You need to sign into your [Amazon S3 console](http://aws.amazon.com/s3/), create a S3 bucket and write the bucket's name to the field `S3:Bucket` in `conf/ufs.yml`. To use other under storage systems, configure the field `Type` and the corresponding configurations in `conf/ufs.yml`.
-
-Now you can launch the Alluxio cluster with your chosen under filesystem in your chosen availability zone by running
-the script under `deploy/vagrant`:
-
-{% include Running-Alluxio-on-EC2/launch-cluster.md %}
-
-Each node of the cluster runs an Alluxio worker, and the `AlluxioMaster` runs the Alluxio master.
-
-## Access the cluster
+### Access the cluster
```

Comment:
Are all the "access the cluster" the same across all sections? Then, we should pull it out to a top level section, so it is not repeated 3 times.

Reply:
there are parts that are the same, but all three sections contain instructions specific to that deployment scenario as well.